src/content/troubleshooting/_index.en.md

```diff
@@ -55,18 +55,27 @@ TBD
 If you are able to connect to remote service by using ServiceIP or globalIp, but not by service name, it is a Service Discovery Issue.
 
 #### Service Discovery not working
-This is good time to familiarize yourself with [ServiceDiscovery Architecture](../architecture/service-discovery/) if you haven't already.
+This is good time to familiarize yourself with [Service Discovery Architecture](../architecture/service-discovery/) if you haven't already.
 
 ##### Check CoreDNS Configuration
 Submariner configures CoreDNS deployment to enable `lighthouse` plugin. Since CoreDNS is the first point of entry for any DNS queries, that is where we will start first. 
 
-First we check if we are running the correct image. We need CoreDNS image with ligthouse enabled.
+First we check if we are running the CoreDNS image with the Lighthouse plugin enabled.
 
 ```kubectl -n kube-system kube-dns describe deployment```
 
-Make sure that `Image` is set to `quay.io/submariner/lighthouse-coredns:<version>` and version is correct. If not, change it to use the Lighthouse image.
+{{% notice info %}}
 
-If image is correct, next we need to check if `lighthouse` plugin is enabled on the CoreDNS in the source cluster i.e. the cluster making the query.
+For Openshift use ```kubectl -n openshift-dns dns describe deployment```
+
+Some deployments may use different name and namespace for DNS service e.g. `coredns`, `core-dns` etc.
```

Comment:
This is not getting rendered properly in the final output. 

![9](https://user-images.githubusercontent.com/6574656/81540749-b5ec7380-938f-11ea-965c-86309c3498d6.jpg)

Reply:
Should work correctly now